### PR TITLE
Provide PractitionerRole role search parameter (FHIR-47115)

### PR DESCRIPTION
**Pull request by dbojicic**
This PR provides fix for [FHIR-47115](https://jira.hl7.org/browse/FHIR-47115).

Changes in: 
- AU Core PractitionerRole profile Notes & Optional Search Parameters
- AU Core Requester & Responder CapabilityStatements
- Change log

### CODE REVIEW

File: input/pagecontent/changes.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ This change log documents the significant updates and resolutions implemented fr
 - Removed the requirement for including an offset in the Patient 'birthdate' search parameter [FHIR-47150](https://jira.hl7.org/browse/FHIR-47150)
 - Updated requirement for AU Core Requester to mandate both system and code values for identifier search parameters on Location, Organization, Practitioner and PractitionerRole [FHIR-46782](https://jira.hl7.org/browse/FHIR-46782)
 - Changed the Location search parameter 'address' from SHALL to SHOULD [FHIR-47107](https://jira.hl7.org/browse/FHIR-47107)
+- Added a PractitionerRole search parameter 'role' as a SHOULD [FHIR-47115](https://jira.hl7.org/browse/FHIR-47115)
```
Comment (JayMurdoch) [inline input/pagecontent/changes.md:15]: Suggest formatting as per suggestion in https://github.com/hl7au/au-fhir-core/pull/281 "Added a PractitionerRole 'role' search parameter...."

**Review by JayMurdoch — CHANGES_REQUESTED** [review]
Minor change required.

File: input/pagecontent/changes.md (hunk 1 above)
Comment (dbojicic) [reply input/pagecontent/changes.md]: Done

**Review by JayMurdoch — APPROVED** [review]

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md [hunk 2]
```diff
@@ -107,3 +113,15 @@ The following search parameters and search parameter combinations **SHOULD** be
...
     *Implementation Notes:* Fetches a bundle of all PractitionerRole resources for the specified specialty ([how to search by token](http://hl7.org/fhir/R4/search.html#token))
 
+1. **SHOULD** support searching using the **[`role`](https://hl7.org/fhir/R4/practitionerrole.html#search)** search parameter:
+    - **SHOULD** support these **[`_include`](http://hl7.org/fhir/R4/search.html#include)** parameters: `PractitionerRole:practitioner`
+
+    `GET [base]/PractitionerRole?role={system|}[code]`
+
+    Example:
+    
+      1. GET [base]/PractitionerRole?specialty=http://snomed.info/sct\|304292004
```
Comment (brettesler-ext) [inline input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md:123]: specialty should be role

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md [hunk 3]
```diff
@@ -107,3 +113,15 @@ The following search parameters and search parameter combinations **SHOULD** be
...
+      1. GET [base]/PractitionerRole?specialty=http://snomed.info/sct\|304292004&amp;_include=PractitionerRole:practitioner
```
Comment (brettesler-ext) [inline input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md:124]: specialty should be role

**Review by brettesler-ext — CHANGES_REQUESTED** [review]
example fix needed

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md (hunk 3 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md]: Fixed

File: input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md (hunk 2 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-practitionerrole-notes.md]: Fixed

**Comment by dbojicic** [comment]
> example fix needed

Fixed

**Review by brettesler-ext — APPROVED** [review]

**Pull request merged**